configs/dgcnn/dgcnn_4xb32-cosine-100e_s3dis-seg-area1.py

```diff
@@ -0,0 +1,36 @@
+_base_ = [
```

Comment:
Please rename this kind of config on different test splits like: dgcnn_4xb32-cosine-100e_s3dis-seg_test-area1.py